### PR TITLE
update lt-LT lang (part 2)

### PR DESCRIPTION
**Pull request by kiprasmel**
hey, here comes the 2nd update.

i'd like to see where in the UI the following strings are used:

- label.average
- label.sum
- label.breakdown
- label.contains
- label.does-not-contain
- label.continue
- label.false
- label.greater-than
- label.greater-than-equals
- label.less-than
- label.less-than-equals
- label.is
- label.is-not
- label.is-not-set
- label.is-set
- label.true
- label.unique

- label.created
- label.created-by

- label.window
- label.day
- label.dropoff
- label.edit-dashboard
- label.insights
- label.leave 
- label.queries
- label.remove
- label.remove-member
- label.select
- label.sessions
- label.team-view-only
- label.team-websites
- label.total
- label.total-records
- label.transfer
- label.transfer-website
- label.url
- label.urls
- label.value
- label.window

- message.no-event-data
- message.transfer-team-website-to-user
- message.triggered-event
- message.viewed-page
- message.visitors-dropped-off

---

and clarifications for the following:

- label.type -- type of what?

- label.none -- plural or singular?
- label.query-parameters -- "query" plural or singular?

- label.regenerate -- does it mean "create a new one, invalidating the old one", or "create another one"?

- difference between label.title vs label.pageTitle?

---

- [ ] further, there's been a regression in the `tracking code` -- it used to translate to just "tracking code", but now the english text contains "tracking code", and then the instructions to setup the tracking code. old translations do not include these instructions, only the "tracking code" translation. could also add a link to the tracker configuration docs: https://umami.is/docs/tracker-configuration

### CODE REVIEW

File: src/lang/lt-LT.json [hunk 1]
```diff
@@ -82,85 +82,85 @@
...
   "label.queries": "Queries",
-  "label.query": "Query",
-  "label.query-parameters": "Query parameters",
+  "label.query": "Užklausa",
+  "label.query-parameters": "Užklausų parametrai",
   "label.realtime": "Realiuoju laiku",
-  "label.referrer": "Referrer",
-  "label.referrers": "Referrers",
+  "label.referrer": "Persiuntėjas (referrer)",
+  "label.referrers": "Persiuntėjai (referrers)",
```
Comment (kiprasmel) [inline src/lang/lt-LT.json:127]: the word "Persiuntėjas" isn't really common in LT lang, so I've kept the english one in `()` to clarify.
tho haven't tested in the UI if looks ok / no overflow

File: src/lang/lt-LT.json [hunk 2]
```diff
@@ -178,60 +178,60 @@
...
+  "message.reset-website": "Kad atstatyti šią svetainę, įrašykite {confirmation} žemiau, kad patvirtintumėte.",
   "message.reset-website-warning": "Visi šios svetainės statistikos duomenys bus ištrinti, bet sekimo kodas išliks nepaliestas.",
   "message.saved": "Sėkmingai išsaugota.",
   "message.share-url": "Tai yra viešai prieinama {target} nuoroda (URL).",
-  "message.team-already-member": "You are already a member of the team.",
-  "message.team-not-found": "Team not found.",
-  "message.team-websites-info": "Websites can be viewed by anyone on the team.",
+  "message.team-already-member": "Jūs jau esate šios komandos narys.",
+  "message.team-not-found": "Komanda nerasta.",
+  "message.team-websites-info": "Bet kas iš šios komandos gali peržiūrėti svetaines.",
```
Comment (kiprasmel) [inline src/lang/lt-LT.json:228]: "websites", as in "websites owned by this team", or "all websites"?

Comment (mikecao) [reply src/lang/lt-LT.json:228]: That message means team websites.

**Comment by kiprasmel** [comment]
@mikecao i've updated the suggestions. now, would it be possible to see the UI parts that I mentioned in the PRs description? then i'd be able to translate the remaining translations.

otherwise the current translations are ready

**Comment by mikecao** [comment]
All the filter and compare labels are used in report parameters. They will show up like this:
image

**Comment by kiprasmel** [comment]
alright, ready to go.

**Review by mikecao — APPROVED** [review]
🎉

**Pull request merged**